src/borg/testsuite/archiver.py

```diff
@@ -79,6 +79,7 @@ def exec_cmd(*args, archiver=None, fork=False, exe=None, **kw):
                 archiver = Archiver()
             archiver.prerun_checks = lambda *args: None
             archiver.exit_code = EXIT_SUCCESS
+            helpers.return_code = EXIT_SUCCESS
```

Comment:
It can't be in archiver. If you recall at #2198, the reason we did this is that we somehow wanted to tweak the return code from another class method (`Archive.restore_attsr()`). We can't do that if we don't have access to the archiver object. So unless archiver is not global, or is not added to the args of the Archive, this can't be done.